src/Bindings/ManualBindings.cpp

```diff
@@ -1329,7 +1329,7 @@ static int tolua_cPluginLua_AddWebTab(lua_State * tolua_S)
 	)
 	{
 		Reference = luaL_ref(tolua_S, LUA_REGISTRYINDEX);
-		Title = ((std::string)tolua_tocppstring(tolua_S, 2, 0));
+		Title = (static_cast<std::string>(tolua_tocppstring(tolua_S, 2, 0)));
```

Comment:
I will add that to the [list](https://github.com/cuberite/cuberite/issues/2377).